README.md

```diff
@@ -33,7 +44,7 @@ as a Nuget package or you can download the source using the instructions above.
 
 ### Prerequisites
```

Comment:
We shouldn't call it Universal--that term now means UWP on Win10. It looks like we are using the term "Windows Runtime universal C#" in our docs.